client/src/components/DataEntry/SignOff.tsx

```diff
@@ -37,7 +37,7 @@ const SignOff = ({ auditBoard, submitSignoff }: IProps) => {
                 label={`Audit Board Member: ${member.name}`}
               >
                 <LabelText htmlFor={`[${i}]`}>Full Name</LabelText>
-                <NameField name={`[${i}]`} />
+                <NameField name={`[${i}]`} id={`[${i}]`} />
```

Comment:
why this change?

Reply:
@jonahkagan Needed to find the input field of 'Full Name' by label, and the reference of the label wasn’t returning anything because the id of the input was not available.